libPhoneNumberGeocodingTests/NBPhoneNumberOfflineGeocoderTest.m

```diff
@@ -389,4 +389,31 @@ - (void)testConvenienceGetDescriptionForNonGeographicalNumber {
                         [self.geocoder descriptionForNumber:self.koreanMobilePhoneNumber]);
 }
 
+- (void)testPerformanceOverAHundredCalls {
```

Comment:
To get the best data for each measure block, we want to make sure we're in the same state for each run. 

So even though each measure block is running the block 100 times to get the aggregate, that first time it's run there is an additional DB connection. Then for the subsequent 9 runs of the measure block, the initial access doesn't have to create the DB connections because setUp isn't called so it's not restored to the original state.

To prevent that, you may need to do an initial access to the DB before entering the measure block.